components/nimbus/android/src/main/java/org/mozilla/experiments/nimbus/Nimbus.kt

```diff
@@ -156,6 +156,26 @@ interface NimbusInterface {
      */
     fun resetTelemetryIdentifiers() = Unit
 
+    /**
+     * Records the `exposure` event in telemetry.
+     *
+     * This is a manual function to accomplish the same purpose as passing `true` as the
+     * `recordExposureEvent` property of the `getVariables` function. It is intended to be used
+     * when requesting feature variables must occur at a different time than the actual user's
+     * exposure to the feature within the app.
```

Comment:
e.g. experimenting on a menu, where menu construction happens at a different time to the user seeing the menu. 

Or `getVariables` is being asked for multiple times, with the same feature, and we want to record exposure only once.

e.g. when `getVariables` is being asked for every keystroke.